pages/AWorkedExample.html

```diff
@@ -186,7 +186,7 @@ <h4>Example: Adding a response to our Sample Exploration</h4>
     Let's make a response to address this. When you finish saving the
     response, it should look like this:
   </p>
-  <img src="static/images/tutorial/updateExplorationNewResponse.png" class="tutorial-image">
+  <img src="static/images/tutorial/updateExplorationNewResponse.png" class="tutorial-image" alt="A box containing the correct answer followed by text that the learner would recieve on inputting the correct answer, along with the next destination the learner will be redirected to.">
```

Comment:
Describe what's visible in the image; this is too vague for someone to understand (since the textual content of the image is important in this case).

receive is misspelled.